docs/cli/cloudctl_oc_cli.md

```diff
@@ -1,8 +1,8 @@
 ---
```

Comment:
This whole document should be reviewed to determine if it belongs in Open Horizon.

Reply:
There are 8 documents in Open Horizon that reference cloudctl and OpenShift.  Let's take the discussion over to  https://github.com/open-horizon/open-horizon.github.io/issues/221 to make decisions about how to remove / rewrite.